backend/src/external_api/cdogs/cdogs.service.ts

```diff
@@ -0,0 +1,192 @@
+import { Inject, Injectable } from "@nestjs/common";
+import { ExternalApiService } from "../external-api-service";
+import axios, { AxiosRequestConfig, AxiosResponse } from "axios";
+import { get, post } from "../../helpers/axios-api";
+import { ConfigurationService } from "../../v1/configuration/configuration.service";
+import { COMPLAINT_TYPE } from "../../types/models/complaints/complaint-type";
+import { CONFIGURATION_CODES } from "../../types/configuration-codes";
+import { constants } from "http2";
+import FormData = require("form-data");
+import fs = require("fs");
+import { join } from "path";
+
+@Injectable()
+export class CdogsService implements ExternalApiService {
+  readonly authApi: string;
+  readonly baseUri: string;
+  readonly clientId: string;
+  readonly clientSecret: string;
+  readonly grantType: string;
+
+  @Inject(ConfigurationService)
+  readonly configService: ConfigurationService;
+
+  constructor() {
+    this.authApi = process.env.COMS_JWT_AUTH_URI;
```

Comment:
I know there was a chat on slack about this, but I feel like this isn't right.  If I understand this correctly, this has a dependency on the COMS Keycloak realm.  If COMS is taken offline, this would end up breaking our ability to work with CDOGS.  

Does this work with the standard realm?

For example, rename this to JWT_AUTH_URI, and use the values https://dev.loginproxy.gov.bc.ca/auth/realms/standard/protocol/openid-connect/token, https://test.loginproxy.gov.bc.ca/auth/realms/standard/protocol/openid-connect/token, or https://loginproxy.gov.bc.ca/auth/realms/standard/protocol/openid-connect/token

Reply:
As per our slack conversation, I can confirm that the url to authorize CDOGS use is correct so we can't re-use the existing JWT_AUTH_URI but at the same time we can't use that url and just change the realm for additional common services